pulser/tests/test_simresults.py

```diff
@@ -105,7 +106,8 @@ def test_expect():
         results.expect([np.array(3)])
     op = [qutip.tensor(qutip.qeye(2),
                        qutip.basis(2, 1)*qutip.basis(2, 0).dag())]
-    assert len(results.expect(op)[0]) == duration
+    # only initial and final times are evaluated
```

Comment:
This comment doesn't make sense, does it? Isn't `number_of_meas = 1000`?

Reply:
I don't remember editing these lines (ever), that's strange... @sebgrijalva ?

Reply:
It might be an unfortunate copy-paste in the wrong window on my side... I'd just delete the comment anyway